### ISSUE TITLE
help me fix small bug please

### ISSUE THREAD

**Issue by nikolasun** [philroberts/FPDB-for-OSX#41]
How do you do. Im use original 0.40.5 version, becouse this work better for me on windows (just i got problem with screen resolution in graph on your version).

Im got small bug on 0.40.5 in "hands viewer" in "hole cards" filter.

Promlem is in mark "all, suited off suited", that just dont work.

im fish in python (and other)), plz tell me, how i can fix))

ps i dont use you version, becouse i got problem with screen resolution, but ive tested so small, plz tell me if i need to use your version, if that to be best on windows too.

pps sorry for my ENG

**Comment by philroberts** [comment]
Yeah I'm pretty sure they don't work in my latest releases either.  Somebody put the buttons in without adding the code to make them do anything.  Fixing this is a very low priority for me unfortunately.

**Comment by nikolasun** [comment]
no problem, just tell me, that a file i need to edit. i do it himself.
ps i mean ill try))
